### DISCUSSION TITLE
Insert for the Life Fitness IC7 Bike

### DISCUSSION THREAD

**Discussion by Fenar91** [doudar/SmartSpin2k#393] [General]
Hello,
is there an insert vor the life fitness ic7 Bike?

Thank u very much.

**Comment by eMadman** [comment]
I just helped a user with an IC7 a few days ago.  I don't know about the fit yet since he hasn't received his kit.  The bike is identical to the now out of production Flywheel bike.  There were some changes to the frame design but the knob is the same.

There are two inserts you can use - I don't know which one is a better fit for your bike.  From the inserts section, grab the FLYWHEEL 54.5mm and 55mm.  One of them will fit better.  Let me know which it is so we can clean up the repo!  
https://github.com/doudar/SmartSpin2k/blob/develop/Hardware/Inserts/FLYWHEEL_54.5mm.STL
https://github.com/doudar/SmartSpin2k/blob/develop/Hardware/Inserts/FLYWHEEL_55mm.STL


The bike mount for this bike is already in the appropriate directory and I'm quite confident it will fit
https://github.com/doudar/SmartSpin2k/blob/develop/Hardware/MODS/Direct%20Mount%20Mod/Bike%20Mount/Life%20Fitness%20IC7.stl

I don't remember what length arm you'll need.  Try the Peloton one.  It's in-between the shorter Schwinn IC4 and Echelon arms.  


I suggest using the direct mount short case
https://github.com/doudar/SmartSpin2k/tree/develop/Hardware/MODS/Direct%20Mount%20Mod/Short%20Case

**Reply by doudar** [reply]
I also recommend using the direct mount mod case for this bike. I think you'll want the ability to pivot the arm down so it can be positioned on the head tube better. 

We're talking about making the direct mount case the default case because there's so many bikes with round top knobs and SS2K needs the extra support. 

Please let us know how it goes! We'd love to see pictures!

**Reply by Fenar91** [reply]
Hello Guys,
i really thank u! I finished one for my schwinn ic4 and the ic7 is for a private customer. I hope it will fit, we will try it. For my ic4 i designed a new kind of case and switchholder. I try to send the link of the pictures.

https://bilderupload.org/bild/750488790-whatsapp-image-2022-10-27
https://bilderupload.org/bild/4c4e88820-whatsapp-image-2022-10-27
https://bilderupload.org/bild/127888827-whatsapp-image-2022-10-27
https://bilderupload.org/bild/32e088831-whatsapp-image-2022-10-27
https://bilderupload.org/bild/299588837-whatsapp-image-2022-10-27

Thank u!
